x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TimeRetentionPolicyConfigTests.java

```diff
@@ -17,7 +18,10 @@
 public class TimeRetentionPolicyConfigTests extends AbstractSerializingTestCase<TimeRetentionPolicyConfig> {
 
     public static TimeRetentionPolicyConfig randomTimeRetentionPolicyConfig() {
-        return new TimeRetentionPolicyConfig(randomAlphaOfLengthBetween(1, 10), new TimeValue(randomLongBetween(60000, Long.MAX_VALUE)));
+        return new TimeRetentionPolicyConfig(
+            randomAlphaOfLengthBetween(1, 10),
+            new TimeValue(randomLongBetween(60000, Long.MAX_VALUE - 1))
```

Comment:
Long max value - 1 = `9223372036854775806L` which, transforming from milliseconds to days is: 

106751991167.30064

`new TimeValue()` assumes that the value will be `ms`. I think this will cause flakiness.